Source/WebKit/ModelProcess/cocoa/ModelProcessModelPlayerProxy.mm

```diff
@@ -169,6 +176,17 @@ static CGFloat effectivePointsPerMeter(CALayer *caLayer)
     return defaultPointsPerMeter;
 }
 
+void ModelProcessModelPlayerProxy::updateBackgroundColor()
+{
+    if (!m_layer)
+        return;
+
+    if (m_backgroundColor.isValid())
+        [m_layer setValue:(__bridge id)cachedCGColor(m_backgroundColor).get() forKeyPath:@"separatedOptions.material.clearColor"];
```

Comment:
what happens if this color has alpha?

Reply:
From my observation so far, it punches through all the way to passthrough, but with constant flickering. Reached out to the relevant folks asking what should I expect, but I don't have a good answer yet. 
But in any case I think showing any level of passthrough is the wrong behavior there.

Reply:
agreed

Reply:
should we blend it with white or something?

Reply:
We already force it to alpha=1 in the backgroundColor setter:
```
m_backgroundColor = color.opaqueColor();
```

Whether we should blend with white instead of forcing alpha=1 is an interesting idea that we can explore down the road! It will be a simple fix.